psi4/src/psi4/cc/ccresponse/scatter.cc

```diff
@@ -291,14 +291,14 @@ void scatter(std::shared_ptr<Molecule> molecule, Options &options, double step,
     // Mass-weighting the co-ordinates //
     // double massi[natom];
     outfile->Printf("\tAtomic Masses for Raman Computation:\n");
-    for (i = 0; i < natom; i++) outfile->Printf("\t%d %12.8f\n", i, molecule->mass(i));
+    for (i = 0; i < natom; i++) outfile->Printf("\t%d %12.8f\n", i, molecule->mass(i, false));
```

Comment:
From my understanding a ghost atom is just a point in space where we are putting basis functions as if an atom were there, but no atom is really there. 

Assuming that is correct mass weighting and computing the center of mass should use zero mass for ghosts.

Reply:
That's exactly right. You might want a non-zero mass if you want consistent frames between real and ghosted fragments (RE:[909 discussion](https://github.com/psi4/psi4/issues/909)). I can see people wanting an un-changing `center_of_mass` during, say, a many body expansion just for `reorient`-like purposes, but for actual property calculations I don't _think_ the ghost masses should contribute to `center_of_mass` or otherwise.